jest.config.js

```diff
@@ -16,23 +16,24 @@ module.exports = {
     '@components/(.*)': '<rootDir>/pkg/rancher-components/src/components/$1',
   },
   transform: {
-    // process js with `babel-jest`
-    '^.+\\.js$':   '<rootDir>/node_modules/babel-jest',
-    // process `*.vue` files with `vue-jest`
-    '.*\\.(vue)$': '<rootDir>/node_modules/@vue/vue2-jest',
-    // process `*.ts` files with `ts-jest`
-    '^.+\\.tsx?$': 'ts-jest'
+    '^.+\\.js$':   '<rootDir>/node_modules/babel-jest', // process js with `babel-jest`
+    '.*\\.(vue)$': '<rootDir>/node_modules/@vue/vue2-jest', // process `*.vue` files with `vue-jest`
+    '^.+\\.tsx?$': 'ts-jest' // process `*.ts` files with `ts-jest`
   },
   snapshotSerializers: ['<rootDir>/node_modules/jest-serializer-vue'],
   collectCoverage:     false,
   collectCoverageFrom: [
     '<rootDir>/shell/**/*.{vue,ts,js}',
     '<rootDir>/pkg/rancher-components/src/components/**/*.{vue,ts,js}',
-    // '!<rootDir>/components/RancherProviderIcon.vue',
+    '!<rootDir>/shell/scripts/',
   ],
-  modulePathIgnorePatterns: ['<rootDir>/cypress/'],
-  coverageDirectory:        '<rootDir>/coverage',
-  coverageReporters:        ['html', 'text'],
+  modulePathIgnorePatterns: [
```

Comment:
This should include docusaurus and stores nows

Reply:
This configuration is only to define where to search the scripts.
It's important to avoid Cypress as it would run them and return error due different syntax.

I add the following as it may improve performances.
```
    '<rootDir>/docusaurus/',
    '<rootDir>/stories/',
    '<rootDir>/shell/scripts/',
```